### ISSUE TITLE
Portal gun has a problem with last shot color

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#136]
```
What steps will reproduce the problem?
1. Select portal gun
2. Shoot orange
3. Shoot blue
4. Shoot blue
5. Shoot blue

What is the expected output? What do you see instead?
The first shot is orange - ok. The second and the third shots are blue - ok. 
The last shot is orange, but blue was selected before shooting.

What version of the product are you using? On what operating system?
Hedgewars 0.9.14 and 0.9.15 have this issue.

Please provide any additional information below.
I've attached a short game replay where you can see this issue.


```

-----
Original issue reported on code.google.com by `dead.maz...@gmail.com` on 28 Dec 2010 at 11:40

Attachments:
* [epic tp fail 9.15.37.hwd](https://storage.googleapis.com/google-code-attachments/hedgewars/issue-136/comment-0/epic tp fail 9.15.37.hwd)

**Comment by GoogleCodeExporter** [comment]
```
I tested it, it seems the last shot from the last ammunition is always orange, 
thats why the issue doesn't apply to Thinking with portals. Make a scheme with 
2 portals ammunition and youll see.
```

-----
Original comment by `charlygh...@gmail.com` on 29 Dec 2010 at 1:59

**Comment by GoogleCodeExporter** [comment]
```
I fixed that in 4725:4cb3f7890fbd
Please test :)
Thank you for reporting ^^
```

-----
Original comment by `sheepyluva` on 30 Dec 2010 at 9:47
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Verified, third teleort deletes the first and is in the same colour as first, 
fourth deletes second and is in the same colour as the second
```

-----
Original comment by `pilarski...@gmail.com` on 29 Nov 2011 at 7:25
* Changed state: **Verified**
